lib/commands/check.js

```diff
@@ -0,0 +1,37 @@
+var path = require('path');
+var log = require('db-migrate-shared').log;
+var assert = require('./helper/assert.js');
+var migrationHook = require('./helper/migration-hook.js');
+
+module.exports = function (internals, config, callback) {
+  migrationHook(internals)
+    .then(function () {
+      var Migrator = require('../migrator.js');
+      var index = require('../../connect');
+
+      if (!internals.argv.count) {
+        internals.argv.count = Number.MAX_VALUE;
+      }
+      index.connect({
+        config: config.getCurrent().settings,
+        internals: internals
+      }, Migrator, function (err, migrator) {
+        if (!assert(err, callback)) return;
+
+        if (internals.locTitle) {
+          migrator.migrationsDir = path.resolve(internals.argv['migrations-dir'],
+            internals.locTitle);
+        } else { migrator.migrationsDir = path.resolve(internals.argv['migrations-dir']); }
+
+        internals.migrationsDir = migrator.migrationsDir;
+
+        migrator.driver.createMigrationsTable(function (err) {
```

Comment:
It seems to me like we could nuke the `createMigrationsTable` call; but it also seems to me like we could've done the same for dry-run with migrations up, so I left in place for parity

Reply:
As soon as we read from the db, we currently need to create it, to avoid erroring out.